src/ruby.rs

```diff
@@ -947,6 +947,22 @@ extern {
     ///     [variable.c](https://github.com/ruby/ruby/blob/v2_5_1/variable.c#L459-L463)
     /// * **2.6:** [ruby.h](https://github.com/ruby/ruby/blob/v2_6_0_preview2/include/ruby/ruby.h#L1808)
     pub fn rb_obj_classname(obj: VALUE) -> *const c_char;
+
+    /// Equivalent of `Kernel#block_given?`.
+    /// Returns 1 (true) if yield would execute a block in the current context, or 0 (false)
+    /// otherwise.
+    ///
+    /// # Safety
+    ///
+    /// No known issues
+    ///
+    /// # Defined In
+    ///
+    /// * **2.3:** [ruby.h](https://github.com/ruby/ruby/blob/v2_3_7/include/ruby/ruby.h#L1830)
+    /// * **2.4:** [ruby.h](https://github.com/ruby/ruby/blob/v2_4_4/include/ruby/ruby.h#L1850)
+    /// * **2.5:** [ruby.h](https://github.com/ruby/ruby/blob/v2_5_1/include/ruby/ruby.h#L1877)
```

Comment:
I think we've also been trying to add at least one link to the code itself. In this case:
https://github.com/ruby/ruby/blob/v2_5_1/eval.c#L834-L843